compose/foundation/foundation/src/androidMain/kotlin/androidx/compose/foundation/text/TextFieldCursor.android.kt

```diff
@@ -0,0 +1,6 @@
+package androidx.compose.foundation.text
+
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+internal actual val DefaultCursorThickness: Dp = 2.dp
```

Comment:
Not sure if it's Android vs. desktop, i.e. ChromeOS may benefit from thinner cursor as well. Maybe just MPP lazy property checking current platform?

Reply:
We should look how native ChromeOs cursor looks like before do that change.

Besides that, lazy property will make API complex

Reply:
My point is that Android vs. Desktop could be not correct way of selecting this property, and it seems to be more dynamic, and more depending on actual device dimensions, not operating system.

Reply:
> more depending on actual device dimensions, not operating system.

That was my initial thought, but I can't prove it, so let's make it platform specific for now.

Reply:
Also, small screens would have 1-2px cursor instead of 2-4px on Windows//Linux/macOs in compare to Android